dll/win32/gdiplus/font.c

```diff
@@ -638,7 +638,15 @@ static INT CALLBACK is_font_installed_proc(const LOGFONTW *elf,
     if (type & RASTER_FONTTYPE)
         return 1;
 
+#ifdef __REACTOS__
+    /* NOTE: This elf is dirty. We have to sanitize it before CreateFontIndirect. */
```

Comment:
I'm not sure I understand. Why is `elf` not a valid LOGFONT that can be passed to CreateFontIndirect? That sounds more like a bug in EnumFontFamilies than anything else.

Reply:
Reverted. Still works.